src/routes/ama.tsx

```diff
@@ -0,0 +1,5 @@
+import {redirect} from "@remix-run/node"
+
+export function loader() {
+    throw redirect("https://github.com/bradgarropy/ama")
```

Comment:
I think you can just `return` the redirect here too and you'll get the same behavior. My thought was throwing would. cancel any other in-flight loaders (if any existed, which they shouldn't based on the other comment). However, I think the same would be true with `return`.

I really just like throwing a `redirect`s, since `redirect`s are "stop what you're doing and get me out of here", similar to throwing an error